pkg/controller/migmigration/backup.go

```diff
@@ -124,6 +124,25 @@ func (t *Task) getStageBackup() (*velero.Backup, error) {
 }
 
 func (t *Task) getPodVolumeBackupsForBackup(backup *velero.Backup) *velero.PodVolumeBackupList {
+	nl := map[string]string{
+		"velero.io/backup-name": backup.Name,
+	}
+
+	client, err := t.getSourceClient()
+	if err != nil {
+		log.Trace(err)
```

Comment:
@JaydipGabani in case of error, we would like to return nil. Currently, the program will continue even if there is an error